dt/Compute/DFG-compute-18.0-3cont-2comp-3cpeh-ipv4-geneve-ceph-nfs-fips

```diff
@@ -0,0 +1,56 @@
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change              |  Date   |
+| -------: | :------------------ | :-----: |
+|     v0.1 | Initial publication | 2023-12-14 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Ceph                                       | vm    | 3      |
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+| Compute   | set-nova-scheduler-filter    | Must have          |
+| Glance    | Ceph/RBD                     | Must have          |
```

Comment:
do you really need glance with ceph/RBD cinder with NFS? What is the idea behind this topology?

Reply:
removed this